javaparser-testing/src/test/resources/com/github/javaparser/bdd/dumping_scenarios.story

```diff
@@ -3,10 +3,19 @@ Scenario: When printing the instantiation we should use the right amount of spac
 Given the class:
 public class A {
     Object foo = new Object();
+	Runnable runnable = ()-> System.out.println("running");
```

Comment:
That's perfect but I guess it would be even better if you copy this scenario and create a new one with your lambda in place of the `Object foo = new Object();`.